frontend/src/models/game.js

```diff
@@ -79,4 +81,29 @@ export class Game {
   resetPlayers() {
     this.players.forEach(player => player.reset());
   }
+
+  initializeAffinities() {
+    this.players.forEach(player => player.affinities.setPlayers(this.players));
+  }
```

Comment:
True, this is where I had it for obvious reasons before.
Unfortunately this comes with massive problems when to assign what and where.
Game initializes with player array and player wants another array players within the affinities setup.
It was a real pain in the arse to get this working without changing too much of testsetup and how the game model is initialized.

Other that that, this change is more than welcome. Thinking about it, I should have done it myself, because I want to include random player type generation with the factory pattern in the future anyway...